.github/workflows/update-terms.yml

```diff
@@ -0,0 +1,81 @@
+
+name: Check for dwc terms
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - 'feature/events'
+#on:
+#  schedule:
+#    - cron: '0 0 1 * *'
+jobs:
+  updateterms:
+    runs-on: ubuntu-latest
+    outputs:
+      file-diff1: ${{ steps.check-file-diff-steps.outputs.file-diff1 }}
+      file-diff2: ${{ steps.check-file-diff-steps.outputs.file-diff2 }}
+      file-diff3: ${{ steps.check-file-diff-steps.outputs.file-diff3 }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+      - name: Install project dependencies
+        run: |
+          python -m pip install poetry
+          poetry install
+      - name: Call script to update terms
+        run: |
+          cd ${{ github.workspace }}
+          poetry run update-terms
+          # cat __init__.py
+          # echo test >> __init__.py
+          # echo test1 >> test2.txt
+          # cat __init__.py
+      - id: check-file-diff-step
+        name: Check file diff
+        run: |
+          echo ${{ github.workspace }}
+          cd ${{ github.workspace }}/src/dwcahandler/dwca/terms
+          echo test >> extension-register.csv
+          echo test >> class-rowtype.csv
+          echo test >> terms.csv
+          diff1=$(git diff --name-only extension-register.csv)
+          echo $diff1
+          diff2=$(git diff --name-only class-rowtype.csv)
+          echo $diff2
+          diff3=$(git diff --name-only terms.csv)
+          echo $diff3
+          {
+            echo "file-diff1=$diff1"
+            echo "file-diff2=$diff2"
+            echo "file-diff3=$diff3"
+           } >> $GITHUB_OUTPUT
+      - name: Create pull request
+        if: ${{ (steps.check-file-diff-step.outputs.file-diff1 != '') || 
+                (steps.check-file-diff-step.outputs.file-diff2 != '') ||
+                (steps.check-file-diff-step.outputs.file-diff3 != '') }}
+        run: |
+          # echo "NOW=$(date +'%Y-%m-%dT%H:%M:%S')"
+          # echo $NOW
+          git checkout -b gbif-extension-update
```

Comment:
Will it be easier to have a time-stamped branch as every time that we have a change we need to merge dev onto this `gbif-extension-update` branch then do the commits and PR.